config/src/config/storage_config.rs

```diff
@@ -65,9 +65,13 @@ pub struct RocksdbConfigs {
     pub ledger_db_config: RocksdbConfig,
     pub state_merkle_db_config: RocksdbConfig,
     // Note: Not ready for production use yet.
+    // TODO(grao): Deprecate this flag and use the split_ledger_db_to_individual_dbs below.
     pub use_state_kv_db: bool,
     // Note: Not ready for production use yet.
     pub use_sharded_state_merkle_db: bool,
+    // Note: Not ready for production use yet.
+    // TODO(grao): Add RocksdbConfig for individual DBs when necessary.
+    pub split_ledger_db_to_individual_dbs: bool,
```

Comment:
nit: is the name better than something short, for example just `split_ledger_db`?

Reply:
Done